app/partials/concept-search.html

```diff
@@ -0,0 +1,80 @@
+<div>
+    <h2>Concept Dictionary Maintenance</h2>
+    <b>Find Concept(s)</b>
+    <div>
+        <div>
+            <span>
+                <span>Find a concept by typing in its name:</span>
+                            <input id="inputNode" type="text" ng-model="query" ng-change="timeoutRefresh()">
+                            <input id="searchButton" name="searchButton" value="Search" type="button" ng-click="refreshResponse()">
+                            <br>
+                            <input id="includeVerbose" type="checkbox" ng-model="enableDescription">
+                            <label for="includeVerbose">Show Details</label>
+                            <br>
+                            <br>
+
+                <span style="display: inline;">
+                    <div align="center">
+
+                        <span ng-show="isUserTyping && query.length > 0"> <b>{{searchNotification}}</b> </span>
+                        <br>
+                        <img ng-show="isUserTyping && query.length > 0" src="../../moduleResources/webservices/rest/js/swagger-ui/dist/images/inprogress.gif" width="64" height="64">
+                        <span ng-show="query.length == 0 && !isUserTyping"><b>{{noSearchInputNotification}}</b></span>
+                        <span ng-show="query.length > 0 && concepts.length == 0 && !isUserTyping"><b>{{resultNotification}}</b></span>
+
+                        <table ng-show="concepts.length > 0 && query.length > 0 && !isUserTyping" border="solid" style="width: 100%">
```

Comment:
Could you please move ng-show conditions to functions in controller? e.g. it would be more readable here if you just called ng-show="hasResults()"